crates/cli/src/common_args.rs

```diff
@@ -27,5 +27,5 @@ pub fn yes() -> Arg {
         .long("yes")
         .short('y')
         .action(SetTrue)
-        .help("Assume \"yes\" as answer to all prompts and run non-interactively")
+        .help("Run non-interactively wherever possible. This will answer \"yes\" to almost all prompts.")
```

Comment:
I tweaked this helptext since this flag really prioritizes "proceed as non-interactively as possible"; in the case of logging in on-demand, that means answering _no_ to "login with spacetimedb.com".

Reply:
Perhaps we should call that example out in the help text?

Reply:
I'm worried that it will be _more_ confusing if we make users think about it too hard, since I think "run non-interactively" is what they want deep down.

I guess it's probably fair to add it.

Reply:
Updated

Reply:
Yeah that's a fair point, but probably good to call out.